docs/examples.rst

```diff
@@ -213,6 +216,137 @@ write can be performed by::
 This will write the rootfs image ``rootfs.ext4`` to the slot ``rootfs.0``. Note
 that this bypasses all update mechanics like hooks, slot status etc.
 
+Example Slot Configurations
+---------------------------
+
+This provides some common examples on how to configure slots in your
+system.conf for different scenarios.
+
+Symmetric A/B Setup
+~~~~~~~~~~~~~~~~~~~
+
+This is the default case when having a fully-redundant root file system
+
+.. code-block:: cfg
+  :emphasize-lines: 3, 6, 8, 11
+
+  [...]
+
+  [slot.rootfs.0]
+  device=/dev/sda2
+  type=ext4
+  bootname=A
+
+  [slot.rootfs.1]
+  device=/dev/sda3
+  type=ext4
+  bootname=B
+
+
+Asymmetric A/Recovery Setup
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case storage is too restricted for a full A/B redundancy setup, an
+asymmetric setup with a dedicated update/recovery slot can be used.
+The recovery slot can be way smaller than the rootfs one as it needs to contain
+only the tools for updating the rootfs slot.
+Because the recovery slot is not meant to be updated in most cases, we can
+manifest this for RAUC by setting the ``readonly=true`` option.
+
+.. code-block:: cfg
+  :emphasize-lines: 3, 6, 7, 9, 12
+
+  [...]
+
+  [slot.recovery.0]
+  device=/dev/sda2
+  type=ext4
+  bootname=R
+  readonly=true
+
+  [slot.rootfs.0]
+  device=/dev/sda3
+  type=ext4
+  bootname=A
+
+Separate Application Partition
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+RAUC allows to have a separate redundant set of slots for the application (or
+other purpose) that have a fixed relation to their corresponding rootfs slots.
+RAUC assures that an update of the entire slot group (rootfs + appfs) is
+atomic.
+
+When defining appfs slots, be sure to set the correct `parent` relation to the
+associated bootable slot.
+
+.. code-block:: cfg
+  :emphasize-lines: 14, 19
+
+  [...]
+
+  [slot.rootfs.0]
+  device=/dev/sda2
+  type=ext4
+  bootname=A
+
+  [slot.rootfs.1]
+  device=/dev/sda3
+  type=ext4
+  bootname=B
+
+  [slot.appfs.0]
+  parent=rootfs.0
+  device=/dev/sda4
+  type=ext4
+
+  [slot.appfs.1]
+  parent=rootfs.1
+  device=/dev/sda5
+  type=ext4
+
+Atomic Bootloader Updates (eMMC)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Updating the Bootloader is also possible with RAUC, despite this is a bit more
+critical than updating the rootfs, as there is no fallback mechanism.
+
+However, depending on the ROM loader it can at least be possible to perform the
+bootloader update atomically.
+The most common example for this is using the two boot partitions of an eMMC
+for atomic bootloader updates which RAUC supports out-of-the-box.
```

Comment:
Should this point to https://rauc.readthedocs.io/en/latest/advanced.html#update-emmc-boot-partitions?

Reply:
:+1:

Reply:
:heavy_check_mark: